app/controllers/sessions_controller.rb

```diff
@@ -7,6 +7,7 @@ def create
     
     if user && user.authenticate(params[:session][:password])
       log_in user
+      params[:session][:remember_me] == "1" ? remember(user) : forget(user)
```

Comment:
Nên đổi thành `"1" == ...` do theo convention của Framgia, nếu trong biểu thức so sánh có hằng số thì nên để trước toán tử. Lỡ nhầm `==` thành `=`  thì có thể biết ngay, còn để thế này thì lỡ để `=` câu lệnh gán sẽ luôn diễn ra và luôn gọi hàm remember.